### ISSUE TITLE
IntelliJ/Jetbrains IDE fails to start with custom docker image

### ISSUE THREAD

**Issue by dmckernanacv** [loft-sh/devpod#300]
**What happened?**
With an existing devcontainer (which works fine with VSCode), attempted to open with IntelliJ Ultimate but failed to start.

**What did you expect to happen instead?**
Expected the IDE to start successfully, it seems there should be some kind of process to ensure all the dependencies required by Jetbrains/Gateway remote system will be met.

Here is the log from the Jetbrains Gateway connection:

<details>
<summary>Jetbrains Gateway log</summary>

```
2023-05-17 08:31:52,247	INFO	uname -sm
	stdout:
	Linux aarch64

2023-05-17 08:31:52,261	INFO	echo $SHELL
	stdout:
	/bin/bash

2023-05-17 08:31:52,382	INFO	uname -sm
	stdout:
	Linux aarch64

2023-05-17 08:31:52,499	INFO	echo $SHELL
	stdout:
	/bin/bash

2023-05-17 08:31:52,647	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ echo\ \$HOME
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	/root

2023-05-17 08:31:52,767	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ echo\ \$XDG_CACHE_HOME
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_

2023-05-17 08:31:52,887	WARN	exit code: 1	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ test\ -f\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_

2023-05-17 08:31:53,008	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ dirname\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	/root/.cache/JetBrains/RemoteDev/remote-dev-worker

2023-05-17 08:31:53,126	WARN	exit code: -1	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ ls\ -la\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker

2023-05-17 08:31:53,149	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ mkdir\ -p\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_

2023-05-17 08:31:53,286	INFO	dd of=/root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3.41227306678416.tmp
	stdout:
	stdout used for binary data
	stderr:
	4736+0 records in
	4736+0 records out
	2424832 bytes (2.4 MB, 2.3 MiB) copied, 0.0803297 s, 30.2 MB/s

2023-05-17 08:31:53,481	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ mv\ -v\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3.41227306678416.tmp\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	renamed '/root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3.41227306678416.tmp' -> '/root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3'

2023-05-17 08:31:53,599	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ chmod\ 755\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_

2023-05-17 08:31:53,717	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ test\ -f\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_

2023-05-17 08:31:53,840	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ test\ -x\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_

2023-05-17 08:31:53,973	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ get-sha256\ --path=/root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3

2023-05-17 08:31:54,097	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ get-path\ --path=cache
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	/root/.cache/JetBrains

2023-05-17 08:31:54,215	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ get-path\ --path=config
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	/root/.config/JetBrains

2023-05-17 08:31:54,336	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ exists\ --path=/root/.cache/JetBrains
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	true

2023-05-17 08:31:54,453	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ exists\ --path=/root/.config/JetBrains
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	false

2023-05-17 08:31:54,575	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ create-dir\ --path=/root/.config/JetBrains
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_

2023-05-17 08:31:54,696	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ lock-support
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_

2023-05-17 08:31:54,816	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ available-memory
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	32828612

2023-05-17 08:31:54,942	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ cpu-count
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	5

2023-05-17 08:31:55,069	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ port-forwarding-test
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	34351

2023-05-17 08:31:55,188	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ readlink\ --path=/workspaces/backstage
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	/workspaces/backstage

2023-05-17 08:31:55,297	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ get-path\ --path=cache
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	/root/.cache/JetBrains

2023-05-17 08:31:55,410	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ get-path\ --path=config
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	/root/.config/JetBrains

2023-05-17 08:31:55,524	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ exists\ --path=/root/.cache/JetBrains
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	true

2023-05-17 08:31:55,643	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ exists\ --path=/root/.config/JetBrains
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	true

2023-05-17 08:31:55,761	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ get-path\ --path=cache
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	/root/.cache/JetBrains

2023-05-17 08:31:55,883	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ get-path\ --path=config
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	/root/.config/JetBrains

2023-05-17 08:31:56,007	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ available-space\ --path=/root/.cache/JetBrains
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	87085367296

2023-05-17 08:31:56,128	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ available-space\ --path=/root/.config/JetBrains
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	87085363200

2023-05-17 08:31:56,259	WARN	exit code: 1	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ host-status\ --ide-path=/home/root/.cache/JetBrains/RemoteDev/dist/intellij\ --project-path=/workspaces/backstage
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	Stdout: 
	Stderr: 
	{"type":"error","errorCode":"CommandError","data":"fork/exec /home/root/.cache/JetBrains/RemoteDev/dist/intellij/bin/remote-dev-server.sh: no such file or directory"}

	[command is repeated 53 more times]

2023-05-17 08:32:57,229	INFO	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ backend-status-alive\ --project-path=/workspaces/backstage
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	false

2023-05-17 08:32:57,353	WARN	exit code: 1	/bin/bash -lc echo\ REMOTE_EXEC_OUTPUT_MARKER_\ \&\&\ /root/.cache/JetBrains/RemoteDev/remote-dev-worker/remote-dev-worker_6d6fd444f51603f0cc5a6f8fe6627b75da5d18d44d77559a9b06693f44b33fd3\ product-code\ --ide-path=/home/root/.cache/JetBrains/RemoteDev/dist/intellij
	stdout:
	REMOTE_EXEC_OUTPUT_MARKER_
	{"type":"error","errorCode":"CommandError","data":"/home/root/.cache/JetBrains/RemoteDev/dist/intellij/build.txt doesn't exist"}


==== FAILURES ====

The following exception failed the deployment
com.jetbrains.gateway.ssh.deploy.DeployException: 

Details:
An error occurred while executing command: 'product-code --ide-path=/home/root/.cache/JetBrains/RemoteDev/dist/intellij'
Exit code: 1
	at com.jetbrains.gateway.ssh.DeployFlowUtil$fullDeployCycleImpl$2.invokeSuspend(DeployFlowUtil.kt:280)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at com.intellij.openapi.rd.util.CoroutineProgressContext$Companion$create$task$1$3.invokeSuspend(BackgroundProgressCoroutineUtil.kt:172)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:284)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
	at com.intellij.openapi.rd.util.CoroutineProgressContext$Companion$create$task$1.invoke(BackgroundProgressCoroutineUtil.kt:168)
	at com.intellij.openapi.rd.util.CoroutineProgressContext$Companion$create$task$1.invoke(BackgroundProgressCoroutineUtil.kt:158)
	at com.intellij.openapi.rd.util.CoroutineProgressContext$Companion$createModal$1$1.run(BackgroundProgressCoroutineUtil.kt:203)
	at com.intellij.openapi.progress.impl.CoreProgressManager.startTask(CoreProgressManager.java:429)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.startTask(ProgressManagerImpl.java:114)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressSynchronously$9(CoreProgressManager.java:513)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$new$0(ProgressRunner.java:84)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:252)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:186)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:604)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:679)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:635)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:603)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:173)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:252)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: com.jetbrains.gateway.ssh.deploy.DeployException: 

Details:
An error occurred while executing command: 'product-code --ide-path=/home/root/.cache/JetBrains/RemoteDev/dist/intellij'
Exit code: 1
	at com.jetbrains.gateway.ssh.DeployFlowUtil$fullDeployCycleImpl$2.invokeSuspend(DeployFlowUtil.kt:275)
	... 35 more
Caused by: com.jetbrains.gateway.ssh.deploy.DeployException: 

Details:
An error occurred while executing command: 'product-code --ide-path=/home/root/.cache/JetBrains/RemoteDev/dist/intellij'
Exit code: 1
	at com.jetbrains.gateway.ssh.DeployFlowUtil$fullDeployCycleImpl$2.invokeSuspend(DeployFlowUtil.kt:273)
	... 35 more
Caused by: com.jetbrains.gateway.ssh.RemoteCommandException: 

Details:
An error occurred while executing command: 'product-code --ide-path=/home/root/.cache/JetBrains/RemoteDev/dist/intellij'
Exit code: 1
	at com.jetbrains.gateway.ssh.GoHighLevelHostAccessor.anErrorOccurred(GoHighLevelHostAccessor.kt:184)
	at com.jetbrains.gateway.ssh.GoHighLevelHostAccessor.callAndReportError(GoHighLevelHostAccessor.kt:173)
	at com.jetbrains.gateway.ssh.GoHighLevelHostAccessor.access$callAndReportError(GoHighLevelHostAccessor.kt:29)
	at com.jetbrains.gateway.ssh.GoHighLevelHostAccessor$callAndReportError$1.invokeSuspend(GoHighLevelHostAccessor.kt)
	... 35 more



==== ENVIRONMENT ====

INSTALLED PRODUCTS

	


AVAILABLE MEMORY

	31.31GB



==== DIAGNOSTIC ERRORS ====

Collecting fixes for fixer com.jetbrains.gateway.ssh.deploy.fixes.LogDownloadDiagnosticProvider@512985fa failed with an exception: com.jetbrains.gateway.ssh.RemoteCommandException: 

Details:
An error occurred while executing command: 'product-code --ide-path=/home/root/.cache/JetBrains/RemoteDev/dist/intellij'
Exit code: 1
com.jetbrains.gateway.ssh.RemoteCommandException: 

Details:
An error occurred while executing command: 'product-code --ide-path=/home/root/.cache/JetBrains/RemoteDev/dist/intellij'
Exit code: 1
	at com.jetbrains.gateway.ssh.GoHighLevelHostAccessor.anErrorOccurred(GoHighLevelHostAccessor.kt:184)
	at com.jetbrains.gateway.ssh.GoHighLevelHostAccessor.callAndReportError(GoHighLevelHostAccessor.kt:173)
	at com.jetbrains.gateway.ssh.GoHighLevelHostAccessor.access$callAndReportError(GoHighLevelHostAccessor.kt:29)
	at com.jetbrains.gateway.ssh.GoHighLevelHostAccessor$callAndReportError$1.invokeSuspend(GoHighLevelHostAccessor.kt)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at com.intellij.openapi.rd.util.CoroutineProgressContext$Companion$create$task$1$3.invokeSuspend(BackgroundProgressCoroutineUtil.kt:172)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
	at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:284)
	at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59)
	at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source)
	at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38)
	at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source)
	at com.intellij.openapi.rd.util.CoroutineProgressContext$Companion$create$task$1.invoke(BackgroundProgressCoroutineUtil.kt:168)
	at com.intellij.openapi.rd.util.CoroutineProgressContext$Companion$create$task$1.invoke(BackgroundProgressCoroutineUtil.kt:158)
	at com.intellij.openapi.rd.util.CoroutineProgressContext$Companion$createModal$1$1.run(BackgroundProgressCoroutineUtil.kt:203)
	at com.intellij.openapi.progress.impl.CoreProgressManager.startTask(CoreProgressManager.java:429)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.startTask(ProgressManagerImpl.java:114)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcessWithProgressSynchronously$9(CoreProgressManager.java:513)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$new$0(ProgressRunner.java:84)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$3(ProgressRunner.java:252)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:186)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:604)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:679)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:635)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:603)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:173)
	at com.intellij.openapi.progress.impl.ProgressRunner.lambda$submit$4(ProgressRunner.java:252)
	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:702)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1$1.run(Executors.java:699)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.util.concurrent.Executors$PrivilegedThreadFactory$1.run(Executors.java:699)
	at java.base/java.lang.Thread.run(Thread.java:833)


Could not get host jstack: 

Details:
An error occurred while executing command: 'get-jstack --ide-path=/home/root/.cache/JetBrains/RemoteDev/dist/intellij --project-path=/workspaces/backstage'
Exit code: 1
```

</details>

Looking through that, it seems Jetbrains may be expecting there to be a `/home/root` directory (not totally sure what is going on there, perhaps because the container is running as root, the home folder is expected to exist?) however the actual home directory for root is `/root` so this may be an incorrect assumption on Jetbrains side.

**I understand this is an issue on Jetbrains side, however, given that DevPod is already injecting things, maybe figuring out where the root home dir is and ensuring that a symbolic link to it exists at** `/home/root` isn't out of the question?

I was able to confirm that a symbolic link at `/home/root` pointing to `/root` fixes the connection.

I also confirmed that using a non-root user did NOT fix the problem, however there was no error, Gateway would just silently kick me out to the connection screen.

**How can we reproduce the bug?** (as minimally and precisely as possible)

My `devcontainer.json`:

```
{
  "build": {
    "dockerfile": "Dockerfile"
  },
  "name": "devpod-test"
}
```

My `Dockerfile`:

```
FROM node:16-bullseye-slim

RUN apt-get update && \
    apt-get install -y --no-install-recommends git openssh-client libsqlite3-dev python python3 python3-pip cmake g++ build-essential && \
    pip3 install mkdocs-techdocs-core==1.0.2 && \
    yarn config set python /usr/bin/python3
```

**Local Environment:**

* DevPod Version: 0.1.2 (btw there is no `devpod --version`, just `devpod version`)
* Operating System: mac
* ARCH of the OS: ARM64

**DevPod Provider:**

* Local/remote provider: docker

**Anything else we need to know?**

**Comment by FabianKramm** [comment]
@dmckernanacv thanks for creating this issue! We will take a look and report back!

**Comment by FabianKramm** [comment]
@dmckernanacv should be available in v0.1.5, feel free to reopen if you feel this fixes it not properly

**Comment by dmckernanacv** [comment]
@FabianKramm thank you for the quick turn-around, unfortunately the fix doesn't seem to be working.

I've updated to 0.1.5 and attempted to re-create my container which originally presented the issue and am still seeing the same error from IntelliJ.

How can I check the DevPod logs? I see the output from the create command but it quickly disappears before I can review it.

I checked the `DevPod.log` within the macOS `Console` application, and there doesn't appear to be any errors. 
I would have expected one based on [this line](https://github.com/loft-sh/devpod/pull/316/files#diff-0d4173762ec4380a23db3595d2e88f56010f5cacd588eecbf2c969cae86e2f5cR41) from the fix.
